frontend/src/presentation/components/Layout.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { Layout as AntLayout, Menu } from "antd";
+import { Link } from "react-router-dom";
+
+const { Header, Footer, Content } = AntLayout;
+
+const Layout: React.FC<{ children: React.ReactNode }> = ({ children }) => {
+	const menuItems = [
+		{
+			key: "1",
+			label: <Link to="/workspace">Workspace</Link>,
+		},
+	];
+
+	return (
+		<AntLayout style={layoutStyle}>
+			<Header style={headerStyle}>
+				<div style={logoStyle}>My App</div>
+				<Menu
+					theme="light"
+					mode="horizontal"
+					defaultSelectedKeys={["1"]}
+					style={menuStyle}
+					items={menuItems}
+				/>
+			</Header>
+			<Content style={contentStyle}>
+				<div style={glassContainerStyle}>{children}</div>
+			</Content>
+			<Footer style={footerStyle}>
+				©2024 CS3219-Team25 - All Rights Reserved
+			</Footer>
+		</AntLayout>
+	);
+};
+
+export default Layout;
+
+const layoutStyle: React.CSSProperties = {
```

Comment:
Since you used .module.css for all the other components, should we use that here too?